### PR TITLE
add minutes from 17th Nov and tidying files

### PR DESCRIPTION
**Pull request by hdrdavies**

### CODE REVIEW

File: current-structure.md [hunk 1]
```diff
@@ -15,9 +18,9 @@ Try to organise who will be running the next meetup on the Monday **before** so
 * Introduce people to gitter channel
 
 **6.15pm**
-* Give an initial presentation
+* Give an initial presentation **explaining about the course and encourage people to apply**
```
Comment (nelsonic) [inline current-structure.md:21]: yesterday's explanation was good. a few of the people were aghast when the fact that FAC is "**_free**_" was mentioned in comparison to other coding academies...

**Comment by nelsonic** [comment]
Also, Twitter Content:
- Find out who in the room is using twitter (_get their handles_)
- Take photo of room when full and tag all the people who gave you handles and encourage them to re-tweet/star

**Comment by iteles** [comment]
@nelsonic You should open that as an issue - valid points :+1:

**Comment by nelsonic** [comment]
@iteles yes, un-related to this PR, apologies.

**Pull request merged**